presto-main/src/main/java/com/facebook/presto/metadata/FunctionListBuilder.java

```diff
@@ -62,8 +62,6 @@
 import static com.facebook.presto.util.ImmutableCollectors.toImmutableList;
```

Comment:
This commit should be squashed into the previous one. We always remove dead code in the commit that it died in